src/NUnitEngine/nunit.common.tests/nunit.common.tests.csproj

```diff
@@ -12,8 +12,6 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)/../../Nullable.props" />
-
```

Comment:
Why did I bother to all nullability if you remove it again?

Reply:
Turns out this was a case of not having the NUnit.Analyzers and I didn't know that was the trick to make it work as a separate test assembly.